<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KisOnionSkinDialog</class>
 <widget class="QDialog" name="KisOnionSkinDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>849</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Onion skin options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="slidersLayout">
     <property name="spacing">
      <number>0</number>
     </property>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="buttonsLayout">
     <item>
      <layout class="QHBoxLayout" name="hideShowButtonlayout">
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnShowHide">
         <property name="text">
          <string>+</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="lblPrevColor">
           <property name="text">
            <string>Previous frames color:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lblNextColor">
           <property name="text">
            <string>Next frames color</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="KColorButton" name="btnBackwardColor">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="KColorButton" name="btnForwardColor">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="KisDoubleSliderSpinBox" name="doubleTintFactor" native="true">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KColorButton</class>
   <extends>QPushButton</extends>
   <header>kcolorbutton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KisDoubleSliderSpinBox</class>
   <extends>QWidget</extends>
   <header>kis_slider_spin_box.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
